cellprofiler/modules/classifyobjects.py

```diff
@@ -582,8 +582,6 @@ def run_two_measurements(self, workspace):
             nobjects = len(class_1)
             mapping = np.zeros(nobjects+1, int)
             mapping[1:] = np.arange(1, nobjects+1)
-            for i in range(2): 
```

Comment:
Handling display of NaN values is taken care of elsewhere  (a few lines above this code) and this code has an off-by-one error that showed up when I added the test below.